### ISSUE TITLE
Modular recipe is broken

### ISSUE THREAD

**Issue by faustinoaq** [amberframework/recipes#8]
When I use `amber new app -r misc/modular --deps && cd app && shards build app` I get this:

```
Error in src/app.cr:1: while requiring "../config/*"

require "../config/*"
^

in config/application.cr:10: while requiring "../src/modules/**": can't find file '../src/modules/**' relative to '/home/main/Projects/recipes/app/config'

require "../src/modules/**"
^
```

This issue should be fixed because is required by #7

**Comment by faustinoaq** [comment]
I remember I had this issue first time I tried it and I just created the folder `src/modules` to fix it.

I guess we need something like `src/modules/.gitkeep` to include this folder as well

**Comment by damianham** [comment]
I fixed this in #5

**Comment by faustinoaq** [comment]
@damianham #5 doesn't include the updated `.zip` file, although I think #5 is still useful because updates the js bundle, so, I suggest:

1. Merge #9 
2. Fix specs and include them on all current PRs
3. Include the updated `modular.zip` on #5 and merge it
